README.md

```diff
@@ -83,17 +84,18 @@ module "es_alarms" {
 |-----------------------------------------------|-------------|:----:|:-------:|:--------:|
 | `domain_name`                                 | The Elasticserach domain name you want to monitor. | string | - | yes |
 | `cluster_type`                                | The type of cluster, single or multi-node | string | `"single"` | no |
-| `monitor_cluster_status_is_red_periods`      | The number of periods to alert that cluster status is red, raise this to be less noisy | number | `1` | no |
+| `cluster_status_is_red_periods`      | The number of periods to alert that cluster status is red, raise this to be less noisy | number | `1` | no |
 | `alarm_cluster_status_is_yellow_periods`      | The number of periods before triggering the cluster status is yellow, raise this to be less noisy | number | `1` | no |
```

Comment:
Since you try to standardize variable names here, can you also change `alarm_cluster_status_is_yellow_periods` to `cluster_status_is_yellow_periods` and change `alarm_free_storage_space_too_low_periods` to `free_storage_space_too_low_periods`?